wire-ios-system/Source/ZMSLog.swift

```diff
@@ -278,6 +280,20 @@ extension ZMSLog {
         }
     }
 
+
+    //clears tmp directory
+    @objc public static func clearTmpDirectory() {
+        guard let tmpDirectoryPath = tmpDirectory else { return }
+        closeHandle()
+        let manager = FileManager.default
+        try? manager
+            .contentsOfDirectory(at: tmpDirectoryPath, includingPropertiesForKeys: nil, options: .skipsSubdirectoryDescendants)
```

Comment:
Why do we need the option `skipsSubdirectoryDescendants` ?

Reply:
skipsSubdirectoryDescendants reduces iteration into subdirectories. As it is for deleting only top-level directories are enough to fetch.